Source/relay/TourGuide/Items of the Month/2023/Cursed Monkey Paw.ash

```diff
@@ -0,0 +1,271 @@
+            "",
+            !__quest_state["Level 11 Hidden City"].finished,
+            locationAvailable($location[The Hidden Park])
+        ),
+        new Wishable(
+            $item[lion oil],
+            $effect[none],
+            "",
+            $item[mega gem].available_amount() < 1 &&
+                $item[lion oil].available_amount() < 1,
+            locationAvailable($location[Whitey's Grove])
+        ),
+        new Wishable(
+            $item[bird rib],
+            $effect[none],
+            "",
+            $item[mega gem].available_amount() < 1 &&
+                $item[bird rib].available_amount() < 1,
+            locationAvailable($location[Whitey's Grove])
+        ),
+        new Wishable(
+            $item[drum machine],
+            $effect[none],
+            "",
+            get_property_int("desertExploration") < 100 &&
+                $item[drum machine].available_amount() < 1,
+            locationAvailable($location[The Oasis])
+        ),
+        new Wishable(
+            $item[green smoke bomb],
+            $effect[none],
+            "",
+            __quest_state["Level 12"].state_string["Side seemingly fighting for"] != "hippy",
+            !__quest_state["Level 12"].finished &&
+                locationAvailable($location[The Battlefield (Frat Uniform)])
+        ),
+        new Wishable(
+            $item[star chart],
+            $effect[none],
+            "",
+            !__quest_state["Level 13"].state_boolean["Richard's star key used"] &&
+                $item[Richard's star key].available_amount() < 1 &&
+                $item[star chart].available_amount() < 1,
+            locationAvailable($location[The Hole In The Sky])
+        ),
+        new Wishable(
+            $item[none],
+            $effect[Frosty],
+            "init/item/meat for 8-bit",
+            !__quest_state["Level 13"].state_boolean["digital key used"] &&
+                $item[digital key].available_amount() < 1 &&
+                get_property("8BitScore") < 10000,
+            true
+        ),
+        new Wishable(
+            $item[lowercase N],
+            $effect[none],
+            "",
+            !__quest_state["Level 13"].state_boolean["king waiting to be freed"] &&
+                $item[wand of nagamar].available_amount() < 1 &&
+                $item[lowercase N].available_amount() < 1,
+            locationAvailable($location[The Valley of Rof L'm Fao])
+        )
+    };
+
+    Wishable [int] aftercoreWishables = {
+        new Wishable(
+            $item[bag of foreign bribes],
+            $effect[none],
+            "",
+            locationAvailable($location[The Ice Hotel]),
+            true
+        )
+    };
+
+    int monkeyWishesLeft = clampi(5 - get_property_int("_monkeyPawWishesUsed"), 0, 5);
+    string [int] options;
+    if (__misc_state["in run"] && my_path().id != PATH_COMMUNITY_SERVICE) {
+        options.listAppendList(showWishables(inRunWishables));
```

Comment:
okay my one callout here -- this feels like this tile would be enormously long to users who have unlocked a lot and not finished a lot. should we think through filtering criteria or something that forces the list to be a little more compact, like not showing > 8 wishes or something?

Reply:
I agree, this gets fairly big quickly. What do you think about addressing that in a separate follow-up PR? I want to play with some options but I'm inclined not to hold up getting the first version of the tile in based on that. The good news is even though it can get big, it's down in the resources section so it's not going to monopolize the top tasks area or anything.

Reply:
that is fine. i'll approve and merge after i skim it in a few mins